>>> 'Pkg.add("CUDNN")' log
INFO: Cloning cache of CUDNN from git://github.com/JuliaGPU/CUDNN.jl.git
INFO: Installing CUDArt v0.1.3
INFO: Installing CUDNN v0.1.0
INFO: Building CUDArt
nvcc --shared --compiler-options=-fPIC --linker-options=-fPIC wrapcuda.c -o libwrapcuda.so
make: nvcc: Command not found
make: *** [libwrapcuda.so] Error 127
===============================[ ERROR: CUDArt ]================================

LoadError: failed process: Process(`make`, ProcessExited(2)) [2]
while loading /home/vagrant/.julia/v0.4/CUDArt/deps/build.jl, in expression starting on line 30

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: CUDArt had build errors.

 - packages with build errors remain installed in /home/vagrant/.julia/v0.4
 - build the package(s) and all dependencies with `Pkg.build("CUDArt")`
 - build a single package by running its `deps/build.jl` script

================================================================================
INFO: Package database updated

>>> 'Pkg.test("CUDNN")' log
Julia Version 0.4.0-pre+7304
Commit b3a1be5 (2015-09-05 13:59 UTC)
Platform Info:
  System: Linux (x86_64-unknown-linux-gnu)
  CPU: Intel(R) Core(TM) i5-2500K CPU @ 3.30GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Sandybridge)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
INFO: Testing CUDNN
WARNING: uint32(x) is deprecated, use UInt32(x) instead.
 in depwarn at ./deprecated.jl:73
 in uint32 at deprecated.jl:50
 in include at ./boot.jl:260
 in include_from_node1 at ./loading.jl:271
 in include at ./boot.jl:260
 in include_from_node1 at ./loading.jl:271
 in include at ./boot.jl:260
 in include_from_node1 at ./loading.jl:271
 in require at ./loading.jl:210
 in include at ./boot.jl:260
 in include_from_node1 at ./loading.jl:271
 in process_options at ./client.jl:308
 in _start at ./client.jl:411
while loading /home/vagrant/.julia/v0.4/CUDArt/src/../gen-6.5/gen_libcudart_h.jl, in expression starting on line 41
ERROR: LoadError: LoadError: LoadError: CUDA runtime API library cannot be found
 in error at ./error.jl:21
 in include at ./boot.jl:260
 in include_from_node1 at ./loading.jl:271
 in include at ./boot.jl:260
 in include_from_node1 at ./loading.jl:271
 in require at ./loading.jl:210
 in include at ./boot.jl:260
 in include_from_node1 at ./loading.jl:271
 in process_options at ./client.jl:308
 in _start at ./client.jl:411
while loading /home/vagrant/.julia/v0.4/CUDArt/src/libcudart-6.5.jl, in expression starting on line 43
while loading /home/vagrant/.julia/v0.4/CUDArt/src/CUDArt.jl, in expression starting on line 29
while loading /home/vagrant/.julia/v0.4/CUDNN/test/runtests.jl, in expression starting on line 2
================================[ ERROR: CUDNN ]================================

failed process: Process(`/home/vagrant/julia/bin/julia --check-bounds=yes --code-coverage=none --color=no /home/vagrant/.julia/v0.4/CUDNN/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
ERROR: CUDNN had test errors
 in error at ./error.jl:21
 [inlined code] from pkg/entry.jl:753
 in __test#412__ at no file:0
 in test at no file
 in anonymous at pkg/dir.jl:31
 in cd at file.jl:22
 [inlined code] from pkg/dir.jl:31
 in __cd#396__ at no file:0
 in cd at no file
 in __test#420__ at no file
 in test at no file
 in process_options at ./client.jl:284
 in _start at ./client.jl:411

>>> End of log
